proxyd/backend.go

```diff
@@ -739,6 +750,15 @@ func (bg *BackendGroup) Primaries() []*Backend {
 	return primaries
 }
 
+func isValidMultiCallTx(rpcReqs []*RPCReq) bool {
+	for _, r := range rpcReqs {
+		if r.Method == "eth_sendRawTransaction" {
+			return true && len(rpcReqs) == 1
```

Comment:
seems like `&& len(rpcReqs) == 1` this should be in the line above

Reply:
also you may just want to use the `IsBatched` function

Reply:
Sure I can update this